Automattic/jetpack
# My Jetpack: hide Jetpack Manage menu item

Fixes https://github.com/Automattic/dotcom-forge/issues/5824

## Proposed changes:
We are changing the sidebar navigation as part of the Untangling Calypso project. See pfsHM7-J-p2


* Hide the Jetpack Manage menu item by not calling the code that adds it.

> [!NOTE]
> This PR removes the call and the tests while it still leaves the class introduced in https://github.com/Automattic/jetpack/pull/34353

### Other information:


## Jetpack product discussion
See p1709175739921149/1709107709.312399-slack-C06DN6QQVAQ



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On an atomic site and a Jetpack site connected to WordPress.com
* Go to the admin dashboard and verify the "Jetpack > Jetpack Manage" menu item does not appear in the sidebar.

### Jetpack

|BEFORE|AFTER| 
|-|-|
|<img width="324" alt="Screenshot 2567-02-29 at 16 51 07" src="https://github.com/Automattic/jetpack/assets/1881481/070bad36-9bf7-47c9-848a-0c3fcefa1b6c">|<img width="328" alt="Screenshot 2567-02-29 at 16 48 57" src="https://github.com/Automattic/jetpack/assets/1881481/cc1677b9-7781-4cdc-8f02-0219bbce4735">|

### Atomic
|BEFORE|AFTER| 
|-|-|
|<img width="326" alt="Screenshot 2567-02-29 at 16 28 34" src="https://github.com/Automattic/jetpack/assets/1881481/9360290a-8f23-46a2-b966-fa816e201945">|<img width="326" alt="Screenshot 2567-02-29 at 16 26 11" src="https://github.com/Automattic/jetpack/assets/1881481/cb5a8f14-4778-4ce1-a196-794b58cbb4ef">|
